### ISSUE TITLE
Deprecate (celtic knot) Logo - replace with summon circle

### ISSUE THREAD

**Issue by drllau** [lexDAO/LexDAO-Design-Assets#1]
# Problem
The early logo (correct me if wrong) was pre-2019 obtained using a "free"  picture site which only gave **personal usage permission**, therefore not suited to a **protected trademark**.

# Solution
Progressively replace with in-house placeholder ("summoniing circle" pending a long-term plan on the direction of the UNA and community [marks](https://en.wikipedia.org/wiki/Certification_mark) in relation to education case studies and LexCorpus by LexDAO ([R](https://tsdr.uspto.gov/#caseNumber=97828729&caseSearchType=US_APPLICATION&caseType=DEFAULT&searchType=statusSearch)). 

# What now

1. rearrange directory to be assets/deprecated and assets/current
2. move old celtic knot along with comments on why
3. tag using Dublin Core and ccREL metadata the new media assets
4. ... incorporate suggestions (incl prem [alt](https://discord.com/channels/682960432272506907/938417925994459137/1205350368368660510)) in comments below

# Future changes

Currently :crossed_swords:  LexNews+ :balance_scale: is using a variation on the summon circle with flames of justice enclosed in boxes and the conflict & balance moved to bookend the name LexNews+. Discord allows for
- 50 static emojis 128x128 PNG
- 5 slots for stickers {EOI, RfC, RfQ, CfP, ADM} 320x320 APNG
  - seeking Expression of interest ... usually a task placeholder with bounty attached
  - a document seeking formal endorsement to be temp-checked b4 rotating ... because they may be multi-server, it needs a unique ID so the poll is not shared
  - a formal signal to work for membership that a legal engineering task is present
  - a signal fire to attract specific classes (not just individual)
  - a generic call for administrative action ... this is when the specific person is not known or ombudsman is MIA, and lexy (our LexDAO bot) will come running if we have a ambassaDAO relationship with that server.

+10/15/30 stickers for 2/7/14 boosts respectively

**Comment by drllau** [comment]
Important but **non-urgent** task is to [remove from the market](https://www.teepublic.com/t-shirt/27073104-lexdao-simple) all the deprecated logos

**Comment by cimplylimited** [comment]
Agree.  Have gone in and removed a significant number of these celtic knots from anywhere I can find them.  I've also gone in and updated lexdao.coop to lexdao.org wherever possible.  There are some outstanding locations such as old DAO platforms including Aragon/Moloch and who knows that should also be combed.
